CFC_DataCollector/recommender/alternative_trips_pipeline.py

```diff
@@ -0,0 +1,56 @@
+import get_trips
+from common import store_trip_in_db
+#import get_trips
+from trip import E_Mission_Trip
+#import Profiles
+
+def alternative_trips(trip_list):
+    '''
+    store trips to database
+    '''
+    for trip in trip_list:
+        store_trip_in_db(trip)
+    
+
+
+def get_user_trips(user_ID, filter_queries):
+    '''
+    get a list of user trips, using get_trips logic (filter? will produce canonical trip?)
+
+    '''
+    return []
+
+def get_user_ids(filter_queries):
```

Comment:
This will be a part of pretty much every pipeline, right?. Put into a common location?